util/client.js

```diff
@@ -47,7 +46,7 @@ export function openURL(vue, url, name, specs, replace) {
 
 export function tryPostLoginRedirect({ route }) {
   const { redirect, is_popup: isPopup } = route.query;
-  if (redirect && REDIRECT_WHITE_LIST.indexOf(parse(redirect, {}).hostname) !== -1) {
+  if (redirect && !isValidRedirectUrl(redirect)) {
```

Comment:
Is the condition flipped? `isValidRedirectUrl` should be true

Reply:
[7369587](https://github.com/likecoin/like-co/pull/1742/commits/736958708167406d0003c08e94e1da4acfbcccce)